frontend/src/MyProfile/MyProfile.jsx

```diff
@@ -18,11 +43,14 @@ export default function MyProfile() {
               <source src="../images/hero.mp4" type="video/mp4" />
             </video>
             <div className="hero-content">
-              <h1>HEY {user.username}!</h1>
+              {user ? <h1>HEY {user.username} !</h1>: ''}
```

Comment:
nit: should we use sentence casing for the content on the app, as opposed to all capitalized?

Reply:
I editted this in my next commit.